ner-search/ner_ranker.py

```diff
@@ -0,0 +1,95 @@
+from collections import namedtuple
+from enum import Enum
+from typing import Dict, List, Optional
+
+import spacy
+from jina.executors.rankers import Match2DocRanker
+
+NEREntity = namedtuple('NEREntity', 'value type')
```

Comment:
It's a matter of taste, but I find `NEREntity = namedtuple('NEREntity', ('value', 'type'))` better readable

Reply:
ah yes, good point